engine/src/flutter/lib/web_ui/lib/src/engine/layers.dart

```diff
@@ -777,7 +956,7 @@ class PlatformViewStyling {
   final PlatformViewClip clip;
 
   ui.Rect mapLocalToGlobal(ui.Rect rect) {
-    return position.mapLocalToGlobal(rect.intersect(clip.outerRect));
+    return position.mapLocalToGlobal(rect).intersect(clip.outerRect);
```

Comment:
Is this the actual bug fix? I'm afraid it's hard to spot due to being buried between hundreds of unrelated changes. Would you consider splitting this into 2 PRs?

Reply:
Yes, this is the bug fix. I'll try to split this up into two PRs.